src/domObject.js

```diff
@@ -20,18 +20,28 @@ export default class UI {
 
   displayData(leadersArray = this.leadersArray) {
     const leadersList = document.querySelector('.leaders-list');
+    const listHeader = document.querySelector('.list-header');
     const displayData = [];
     for (let i = leadersArray.length - 1; i >= 0; i -= 1) { // recent score at top
       const displayDataItem = UI.displayOnDom(leadersArray[i]);
       displayData.push(displayDataItem);
     }
+    if (listHeader.classList.contains('hide')) {
+      listHeader.classList.remove('hide');
+    }
     leadersList.innerHTML = (displayData).join(' ');
   }
 
   static displayOnDom(item) {
     const displayHTML = `
-    <p><strong>${item.user}</strong>: ${item.score}</p>
-    `;
+    <div class="player-list flex">
+      <div class="player-name">
+        <p><strong>${item.user}</strong>:</p>
+      </div>
+      <div class="player-score">
+        <p> ${item.score}</p>
+      </div>
+    </div>`;
     return displayHTML;
   }
 
```

Comment:
- [ **OPTIONAL** ] `README.md` file: I think it would be better if you also add the commands of creating the optimized production after the user has installed the dependencies and how to run your app locally after it has been cloned. Also remember to correct the typing error on command to access the cloned directory(**Good: `cd Leaderboard`**, **Bad: `cd To-do-list`** ). 
 i.e. you can update the `## Get into development` sub section to some thing like this:
```
$ cd Leaderboard

$ npm init -y

$ npm install (install dependencies)

$ npm run build (create the optimized for production)

$ npm  start (run the app locallly)
```
I really suggest you implement the above changes before merging your PR to `dev` to make your documentation more professional